src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -14,9 +14,6 @@
     <AssemblyVersion Condition="'$(TargetGroup)'=='net46'">4.0.0.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PackageTargetFramework Condition="'$(TargetGroup)'==''">netstandard1.4</PackageTargetFramework>
-    <PackageTargetRuntime Condition="'$(TargetsWindows)' == 'true' and '$(TargetGroup)' == ''">win7</PackageTargetRuntime>
-    <PackageTargetRuntime Condition="'$(TargetsLinux)' == 'true' and '$(TargetGroup)' == ''">linux</PackageTargetRuntime>
-    <PackageTargetRuntime Condition="'$(TargetsOSX)' == 'true' and '$(TargetGroup)' == ''">osx.10.10</PackageTargetRuntime>
```

Comment:
I've actually standardized on the version-less RID for all of the IL.  This is more inline with the other distros.  The IL itself isn't version dependent, the shims are.